### PR TITLE
[docs] User concept section improvements

### PR DESCRIPTION
**Pull request by Harshg999**
## What changes were proposed in this pull request?

- Removed the release number confusion
- Better explanation for sharing section along with addition of `public links/gist sharing` and beta `share to slack` feature

### CODE REVIEW

**Comment by Harshg999** [comment]
Fixes #1932  and #1933

File: docs/docs-site/content/user/concept/_index.md [hunk 1]
```diff
@@ -66,6 +66,14 @@ Shared documents will show-up with a little blue icon.
 
 ![Sharing](https://cdn.gethue.com/uploads/2019/04/editor_sharing.png)
 
+Along with document sharing, there is another query sharing option via links which is quicker and easier - ![Public Links/Gist Sharing](https://gethue.com/blog/2020-03-04-datawarehouse-database-sql-collaboration-and-sharing-with-link-and-gist/)
+
+### Share to Slack
+Currently in **Beta**
+
+This integration increases the collaboration with others via Slack. It expands the ability to share query links or gists to the desired slack channels which then unfurls in a rich preview for other members to view.
```
Comment (romainr) [inline docs/docs-site/content/user/concept/_index.md:74]: slack --> Slack

File: docs/docs-site/content/user/concept/_index.md [hunk 2]
```diff
@@ -66,6 +66,14 @@ Shared documents will show-up with a little blue icon.
 
 ![Sharing](https://cdn.gethue.com/uploads/2019/04/editor_sharing.png)
 
+Along with document sharing, there is another query sharing option via links which is quicker and easier - ![Public Links/Gist Sharing](https://gethue.com/blog/2020-03-04-datawarehouse-database-sql-collaboration-and-sharing-with-link-and-gist/)
```
Comment (romainr) [inline docs/docs-site/content/user/concept/_index.md:69]: Less generic and content info? (i.e. moving content from the post here and not pointing to the blog post)

- Public link: quickly sharing parameterized saved reports, customer analyses links
- Gist: quickly share SQL snippets, with a direct link to the SQL editor

... etc

and images?

File: docs/docs-site/content/user/concept/_index.md (hunk 1 above)
Comment (Harshg999) [reply docs/docs-site/content/user/concept/_index.md:74]: Currently just gave a big picture for the beta, this can be further explained after v1 @romainr

Comment (romainr) [reply docs/docs-site/content/user/concept/_index.md:74]: I know, my main comment is below for the other section

File: docs/docs-site/content/user/concept/_index.md (hunk 2 above)
Comment (Harshg999) [reply docs/docs-site/content/user/concept/_index.md]: Added the content from blog post

File: docs/docs-site/content/user/concept/_index.md [hunk 3]
```diff
@@ -66,6 +66,56 @@ Shared documents will show-up with a little blue icon.
...
+
+Just paste the link in the Slack channel and users will get a mini preview:
+
+![Gist in Slack](https://cdn.gethue.com/uploads/2020/03/editor_gist_slack.png)
+
+Clicking on the link will open-up the SQL selection:
+
+![Click Gist](https://cdn.gethue.com/uploads/2020/03/editor_gist_open_presentation_mode.png)
+
+### Share to Slack
```
Comment (romainr) [inline docs/docs-site/content/user/concept/_index.md:113]: nit: probably its own section, but let's see later!

#### Slack

**Review by romainr — APPROVED** [review]
Nice!

**Pull request merged**